packages/vc-verification/src/issuer-def-resolver.ts

```diff
@@ -0,0 +1,44 @@
+import { IIssuerDefinition } from '@energyweb/credential-governance';
+import { providers } from 'ethers';
+import { RoleDefinitionResolverV2__factory } from '@energyweb/credential-governance/ethers/factories/RoleDefinitionResolverV2__factory';
+import { RoleDefinitionResolverV2 } from '@energyweb/credential-governance/ethers/RoleDefinitionResolverV2';
+
+/**
+ * An interface for Resolution of issuers for a namespace
+ */
+export interface IssuerDefinitionResolver {
+  /**
+   * @param namespace
+   * @returns IIssuerDefinition for the given namespace
+   */
+  getRoleIssuerDefinition(namespace: string): Promise<IIssuerDefinition>;
```

Comment:
Maybe shorten it to `getIssuerDefinition`. More concise and more generic (as this code could conceivably be used for credentials other than role credentials) 
```suggestion
  getIssuerDefinition(namespace: string): Promise<IIssuerDefinition>;
```